core/doc_data.cpp

```diff
@@ -152,9 +152,10 @@ void DocData::method_doc_from_methodinfo(DocData::MethodDoc &p_method, const Met
 
 	return_doc_from_retinfo(p_method, p_methodinfo.return_val);
 
-	for (int i = 0; i < p_methodinfo.arguments.size(); i++) {
+	int i = 0;
```

Comment:
These are to allow the use of the `i` variable, and unfortunately c++17 doesn't allow initializing multiple variables of different types in the `for` loop